src/disc-golf-events/DiscGolfEventsComponent.js

```diff
@@ -29,46 +31,151 @@ const DiscGolfEventsComponent = () => {
     fetchEvents();
   }, [valueToOrderBy, orderDirection]);
 
-  const fetchEvents = () => {
-    let url = `${config.discGolfServiceUrl}/public/events`;
-    if (valueToOrderBy) {
-      url += "?valueToOrderBy=" + valueToOrderBy;
-      if (orderDirection) {
-        url += "&orderDirection=" + orderDirection.toUpperCase();
+  const fetchEvents = async () => {
+    try {
+      let url = `http://localhost:24001/api/disc-golf-service/public/events`;
+      if (valueToOrderBy) {
+        url += `?valueToOrderBy=${valueToOrderBy}`;
+        if (orderDirection) {
+          url += `&orderDirection=${orderDirection.toUpperCase()}`;
+        }
       }
+
+      const response = await fetch(url, {
+        method: "GET",
```

Comment:
Here are a lot of changes to GET public events that are not really needed. We don't need to specify method GET as it is default in `fetch`. Also we don't need to specify Content-type, as we don't send any content.
I think the only difference is added setError, but it looks like a lot has changed in this section.